### DISCUSSION TITLE
IBM Plex sans missing glyphs

### DISCUSSION THREAD

**Discussion by raunaqpatel** [IBM/plex#358] [Q&A] [answered]
When I try to reference IBM Plex Sans in my web project through either Google Fonts or by downloading the files from repo. Why cant I access all glyphs like shown on the website like Dotted zero, or slashed zero? 
Does it only come with IBM Plex Mono or something? 

https://github.com/google/fonts/issues/2829

**Comment by kenmcd** [comment, marked as answer]
IBM Plex Sans from release v5.1.3 (2020-09-01) has the features you want.
Slashed zero is available via the OpenType features _Slashed Zero_ (`zero`) and _Stylistic Set 3_ (`ss03`).
Dotted zero is available via the OpenType feature _Stylistic Set 4_ (`ss04`).

**Reply by raunaqpatel** [reply]
Thank you.
So sad that on google fonts IBM Plex is not updated in last 2 years.
Its still at version v2.001 there and I just made an issue to update the font to v5.1.3
https://github.com/google/fonts/issues/2954

**Reply by kenmcd** [reply]
Note: the release versions do not match the font versions included in the release.
Which can be a bit confusing.
For example in release v5.1.3 the IBM Plex Sans is version 3.2, revision v3.002.
So that revision number v3.002 is comparable to the GF v2.001.

Note: GF has a habit of updating fonts and not changing the version number.
They call this a "hotfix".
So when I download fonts from GF I always put the date in the ZIP name.
That way I can save the different versions.
